apps/site/tsconfig.json

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
is it something that we can `"extends": "@faststore/shared/tsconfig.json"`? I notice some differences, though.

Reply:
then we will create a configuration, similar to eslint, which must be shared between projects